args.go

```diff
@@ -99,3 +75,18 @@ func RangeArgs(min int, max int) PositionalArgs {
 		return nil
 	}
 }
+
+// ExactValidArgs returns an error if there are not exactly N positional args OR
+// there are any positional args that are not in the `ValidArgs` field of `Command`
+//
+// Deprecated: now `ExactArgs` honours `ValidArgs`, when defined and not empty
```

Comment:
Typo: "honours" assume should be honors.

Reply:
Isn't it a difference between british (honour) and american (honor)?

Reply:
Haha, well learn something new everyday.  Indeed it is. 
Honor is the preferred spelling in American English.  Are there other places in the ReadMe that use British variation then i don't mind. Otherwise please update it since these comments become part of the GoDoc and should have a certain continuity to them.